bookkeeper-server/src/main/java/org/apache/bookkeeper/client/LedgerCreateOp.java

```diff
@@ -261,15 +269,36 @@ public CreateBuilder withDigestType(org.apache.bookkeeper.client.api.DigestType
         }
 
         @Override
+        public CreateBuilder withLedgerType(LedgerType ledgerType) {
+            this.builderLedgerType = ledgerType;
+            return this;
+        }
+
+        @Override
         public CreateAdvBuilder makeAdv() {
             return new CreateAdvBuilderImpl(this);
         }
 
         private boolean validate() {
+            if (builderLedgerType == null) {
+                LOG.error("invalid null builderLedgerType");
+                return false;
+            }
+
+            if (bk.getConf().getUseV2WireProtocol() && builderLedgerType != LedgerType.FORCE_ON_JOURNAL) {
+                LOG.error("invalid builderLedgerType {} while using v2 wire protocol");
+                return false;
+            }
+
             if (builderWriteQuorumSize > builderEnsembleSize) {
                 LOG.error("invalid writeQuorumSize {} > ensembleSize {}", builderWriteQuorumSize, builderEnsembleSize);
                 return false;
             }
+            if (builderLedgerType.equals(LedgerType.FORCE_DEFERRED_ON_JOURNAL) && builderWriteQuorumSize != builderEnsembleSize) {
+                LOG.error("invalid writeQuorumSize {} != ensembleSize {} for FORCE_DEFERRED_ON_JOURNAL ledger. Striping is not allowed",
+                    builderWriteQuorumSize, builderEnsembleSize);
```

Comment:
I know we had some conversation on this. Did we decide striping is not allowed? Do we know it is because of any known consistency issues??

Reply:
I think that in the latest discussions and the latest version of the document we decided to send force requests to all of the bookies. So actually there should be no problem with striping.

@sijie  @ivankelly

Reply:
@jvrao @eolivelli -- I don't think we disallow striping.

Reply:
enabled striping